scripts/check_samples.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import os
+import pathlib
+import re
+import subprocess
+import sys
+import time
+
+
+def strip(text, suffix):
+    if suffix and text.endswith(suffix):
+        return text[:-len(suffix)]
+    return text
+
+def get_expected_samples(crd_path):
+    # For some reason it's 2-5x faster to do this with yq than it is to do it with pyyaml
+    result = subprocess.run(['yq', 'e', '{"name": .spec.names.singular, "group": .spec.group, "version": .spec.versions[] | select(.name != "*storage").name}', str(crd)], capture_output=True, text=True)
```

Comment:
It looks like this is using the global variable `crd` rather than the `crd_path` passed in?

Reply:
Also, could you add a comment here with an example of the output of the `yq` command? I'm not great with the syntax.

Reply:
Fixed and fixed.

I'm actually not sure how it worked before... lol.